integration-tests/excoveralls/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM elixir:1.4.2
+
+RUN curl -O https://storage.googleapis.com/golang/go1.8.linux-amd64.tar.gz
+RUN tar -xvf go1.8.linux-amd64.tar.gz
+RUN mv go /usr/local
+
+ENV PATH $PATH:/usr/local/go/bin
+RUN go version
+
+ENV GOPATH /go
+RUN mkdir $GOPATH
+ENV PATH $PATH:/go/bin
+
+ENV CCTR=$GOPATH/src/github.com/codeclimate/test-reporter
+RUN mkdir -p $CCTR
+WORKDIR $CCTR
+COPY . .
+RUN go install -v
+
+RUN git clone https://github.com/ale7714/excoveralls.git
```

Comment:
Should this be hosted on the `codeclimate` account?

Reply:
gonna go with `codecliamte-testing`, since we're only this for testing not really any development.